packages/flutter/lib/src/services/font_loader.dart

```diff
@@ -62,7 +66,19 @@ class FontLoader {
             (Uint8List list) => loadFont(list, family)
         )
     );
-    return Future.wait(loadFutures.toList());
+    return Future.wait(loadFutures.toList()).then((List<void> futures) async {
+      if (ServicesBinding.instance != null) {
```

Comment:
Why this check?

Reply:
I thought this method can be run before runapp, where the widget binding is not initialized

Reply:
In that case, you'd just call FlutterWidgetsBinding.ensureInitialize beforehand to make sure the binding is initialized.

Reply:
Although this might end up being removed, FlutterWidgetsBinding is layer violation I think? I think layer it should not have the power to mangling with the binding initialization.
On the other hand, If the binding is not initialize, that mean no one cares about the systemfont has changed, and thus no need to notify anyone.